src/index.js

```diff
@@ -1,41 +1,91 @@
 import './style.css';
-// import './style.css';
 
-const Dynamic = document.querySelector('.todo-maker');
-const todos = [
-    {
-        description : "wake up from bed",
-        completed : false,
-        index: 0,
-    },
-    {
-        description : "take break fast",
-        completed : false,
-        index: 1,
-    },
-    {
-        description : "play apex legends",
-        completed : false,
-        index: 2,
-    },
-    {
-        description : "read my books",
-        completed : false,
-        index: 3,
+ const storage =()=>{
+    const todosArr = JSON.parse(localStorage.getItem('deeds')) || [] ;
+    return todosArr;
+ }
+
+class todo {
+    constructor(description, completed = false, index){
+        this.description = description;
+        this.completed = completed;
+        this.index = index;
     }
-]
+}
+const pushtodo = () =>{
+    const todosArr = storage();
+   const index = todosArr.length;
+   const description = document.getElementById('input').value;
+   const completed = false;
+   const todoList = new todo(description,completed,index);
+   if(description != ''){
+    todosArr.push(todoList);
+    localStorage.setItem('deeds', JSON.stringify(todosArr));
+   }
+}
 
-todos.forEach((deed) => {
-    const content = `
-            <div class="list">
+const Dynamic = document.querySelector('.todo-maker');
+let todos2=() => {
+    const todosArr = storage();
+    Dynamic.innerHTML = '';
+    for (let i = 0; i < todosArr.length; i += 1){
+        let isComplte = todosArr[i].completed == true ? "checked" : "";
+        Dynamic.innerHTML += `
+        <div class="list ${isComplte}">
                 <div class="list-action">
-                    <input type="checkbox">
-                    <p>${deed.description}</p>
+                    <input class ="check" id ="${todosArr[i].index}" ${isComplte} type="checkbox">
+                    <p contenteditable="true"class="p">${todosArr[i].description}</p>
```

Comment:
- [ ] I am unable to edit a task, please implement it
https://www.loom.com/share/1d97dfda97e647468ab0c228ce70106c